documentation/staging/content/release-notes.md

```diff
@@ -8,56 +8,176 @@ draft: false
+  * The time of the first failure, last retry, and the expected time of the next retry can now be easily inferred from Domain resource `.status`.
+
+* Istio compatibility:
+  * Domain configurations are always compatible with [Istio]({{< relref "/managing-domains/accessing-the-domain/istio/istio.md" >}}) and other service mesh products.
+  * Customers are no longer required to enable this support.
+
+##### Upgrade Notes
+
+* This release introduces a new API version for the Domain resource, `weblogic.oracle/v9`.
+  * The previous API version, `weblogic.oracle/v8` introduced with Operator 3.0.0 is deprecated, but is still supported.
+
+* This release also introduces a new custom resource, the `weblogic.oracle/v1` Cluster resource.
+  * This Cluster resource configures a specific WebLogic cluster and allows customers to scale that cluster using the Kubernetes Horizontal Pod Autoscaling (HPA) or similar technologies.
+  * The Cluster resource `.spec` schema is similar to and replaces the former `weblogic.oracle/v8` Domain resource `.spec.clusters[*]` content. 
+  * The Cluster resource `.status` schema mirrors the content of the Domain resource `.status.clusters[*]` content.
+
+* This release simplifies [upgrading the operator]({{< relref "/managing-operators/installation.md" >}}).
+  * The operator added a [schema conversion webhook]({{< relref "/managing-operators/conversion-webhook.md" >}}).
+    * This webhook automates the upgrade of `weblogic.oracle/v8` Domain resources to `weblogic.oracle/v9`.
+    * Customers do not need to take any action to begin using the latest Domain schema; however, the team recommends starting with "v9" Domain resources for any new projects.
+  * Customers who install a single WebLogic Kubernetes Operator per Kubernetes cluster (most common use case) can upgrade directly from any 3.x operator release to 4.0.0. The Helm chart for 4.0.0 will automatically install the schema conversion webhook.
+  * For those customers who install more than one WebLogic Kubernetes Operator in a single Kubernetes cluster:
+    * You must upgrade every operator to at least version 3.4.1 before upgrading any operator to 4.0.0.
+    * As the 4.0.0 Helm chart now also installs a singleton schema conversion webhook that is shared by all 4.0.0 operators in the cluster,
+      you will want to use the `webhookOnly` Helm chart option to install this webhook in its own namespace prior
+      to installing any of the 4.0.0 operators,
+      and also use the `preserveWebhook` Helm chart option with each operator to prevent an operator uninstall from uninstalling the shared webhook.
+  * The operator provides a utility that can be used to convert existing "v8" Domain YAML files to "v9".
+  * Several [Helm chart default values have been changed]({{< ref "#changes-to-helm-chart" >}}). Customers who upgrade their 3.x installations using the `--reuse-values` option during the Helm upgrade will continue to use the values from their original installation.
```

Comment:
[Helm chart default values have been changed]({{< ref "#changes-to-helm-chart" >}})
This reference does not go where you want it to go. Should it go here, #changes-to-operator-helm-chart-configuration-values